### PR TITLE
Domain attributes

### PR DESCRIPTION
**Pull request by carmee-en**
Solution for issue #8 .
Domain and test classes updated to meet new attribute requirements. Tests should pass (although not very exhaustive).

Possible new issues:
- Change attribute 'identifier' to 'id' to match the requirements
- Change 'location' attribute type (currently double[ ])
- Unify coding styles

### CODE REVIEW

File: src/main/java/view/ParticipantsController.java [hunk 1]
```diff
@@ -43,8 +43,12 @@ public String showInfo(Model model, @ModelAttribute AgentLoginData data, HttpSes
         else {
             AgentInfoAdapter adapter = new AgentInfoAdapter(user);
             AgentInfo info = adapter.userToInfo();
-            model.addAttribute("fName", info.getFirstName());
-            model.addAttribute("lName", info.getLastName());
+//            model.addAttribute("fName", info.getFirstName());
+//            model.addAttribute("lName", info.getLastName());
```
Comment (alejgh) [inline src/main/java/view/ParticipantsController.java]: Can you remove these comments please?

**Review by alejgh — CHANGES_REQUESTED** [review]
Looks good to me! It would be great if you could add a simple location test (and maybe also a kind test) to test the new attributes, since we are losing quite a lot of test coverage. You can see here the methods that are not tested right now in the classes you modified:
https://codecov.io/gh/Arquisoft/Agents_i2b/src/64d60a3133f48270808d2934a3673ff74bcf2601/src/main/java/domain/AgentInfo.java
https://codecov.io/gh/Arquisoft/Agents_i2b/src/64d60a3133f48270808d2934a3673ff74bcf2601/src/main/java/domain/Agent.java

You don't need to do exhaustive tests, something simple will do. It's just to avoid losing so much code coverage already in the first merge.

**Review by alejgh — APPROVED** [review]

File: src/main/java/domain/Agent.java [hunk 2]
```diff
@@ -16,69 +19,64 @@
...
+		if (location!=null)
+			builder.append(", location=").append(Arrays.toString(location));
+		builder.append(", email=").append(email);
+		builder.append(", password=").append(password);
+		builder.append(", identifier=").append(identifier);
+		builder.append(", kind=").append(kind).append("]");
+		return builder.toString();
+	}
+
+	@Override
```
Comment (miguelff) [inline src/main/java/domain/Agent.java:77]: Mind formatting the code? ;-)

File: src/test/java/domain_tests/UserTest.java [hunk 3]
```diff
@@ -15,41 +15,30 @@
...
+        damian = new Agent("Damian Rubio", "damian@damianmail.com", "damian123", "222222222B", 2);
+        carmen = new Agent("Carmen", "c@c.com", "cccc", "181818S", 18, 20.48, 156.2);
     }
 
     @Test
-    public void firstNameTest(){
-        Assert.assertEquals("Nicolás", nico.getFirstName());
-        Assert.assertEquals("Jorge", jorge.getFirstName());
-        Assert.assertEquals("Damian", damian.getFirstName());
+    public void nameTest(){
```
Comment (miguelff) [inline src/test/java/domain_tests/UserTest.java:29]: Usually, IDEs have support for automatic generation of getters and setters. For that reason, and even though it increases coverage, testing these code-paths could be omitted as they don't actually exercise logic that is prone to error. 

In words of Jeff Atwood, [The Best Code is No Code At All](https://blog.codinghorror.com/the-best-code-is-no-code-at-all/)

**Comment by miguelff** [comment]
GitHub Pro-tip: If in the PR description you reference an issue prefixed with the words "Closes" or "Fixes", it will automatically close the issue when the PR is merged. 

As an example: A PR containing `This PR fixes #8` in the description, will close the issue #8 when merged.

**Pull request merged**